src/Symfony/Component/Yaml/Tests/YamlTest.php

```diff
@@ -24,6 +25,37 @@ public function testParseAndDump()
         $this->assertEquals($data, $parsed);
     }
 
+    public function testParseWithMultilineQuotes()
+    {
+        $yaml = <<<YAML
+foo:
+  bar: 'baz
+
+'
+  baz: 'Lorem
+
+       ipsum'
+  foobar: 'foobar'
+YAML;
+
+        $this->assertSame(['foo' => ['bar' => "baz\n", 'baz' => "Lorem\nipsum", 'foobar' => 'foobar']], Yaml::parse($yaml));
```

Comment:
Maybe I misunderstanding something, but I'd expected the test to be:

```php
$this->assertSame(['foo' => ['bar' => "baz\n\n", 'baz' => "Lorem\n\nipsum", 'foobar' => 'foobar']], ...)
```

Reply:
Hi @javiereguiluz, 
thanks for your comment. Please have a look at [this commen](https://github.com/symfony/symfony/pull/33119#issuecomment-520350652) there you already mentioned the same an @nicolas-grekas added a yaml parser online tool screenshot, thos is the way yaml works in this cases.